hack/must-gather.sh

```diff
@@ -43,6 +43,21 @@ if [[ "$ocp_cluster" ]]; then
     $K get clusterversion/version -oyaml > $ARTIFACT_DIR/openshift_version.yaml
 fi
 
+echo
+echo "#"
+echo "# OpenShift HyperConverged Resources"
+echo "#"
+echo
+
+HYPERCONVERGED_RESOURCE=$($K get hyperconvergeds.hco.kubevirt.io -oname --ignore-not-found)
```

Comment:
The hyperconverged CRD is [namespaced scoped](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/7f9b6014845b28f13c7662e5a67b4fb41279ba95/config/crd/bases/hco.kubevirt.io_hyperconvergeds.yaml#L18), so this will not work unless the hyperconverged CR is installed in the default namespace (or whatever namespace is the current context). 

Let's add `-A` to this command to get the hyperconverged resource regardless of what namespace it is created in. 

Validated on my OCP system:
```
[core@ocp414-chris ~]$ oc get hyperconvergeds.hco.kubevirt.io -oname --ignore-not-found
[core@ocp414-chris ~]$ oc get hyperconvergeds.hco.kubevirt.io -A -oname --ignore-not-found
hyperconverged.hco.kubevirt.io/kubevirt-hyperconverged
```